gradle.properties

```diff
@@ -24,5 +24,13 @@ systemProp.defaultSparkVersions=3.5
 systemProp.knownSparkVersions=3.3,3.4,3.5
 systemProp.defaultScalaVersion=2.12
 systemProp.knownScalaVersions=2.12,2.13
+# enable the Gradle build cache
+org.gradle.caching=true
+# enable Gradle parallel builds
 org.gradle.parallel=true
+# configure only necessary Gradle tasks
+org.gradle.configureondemand=true
+# explicitly disable the configuration cache
+org.gradle.unsafe.configuration-cache=false
```

Comment:
i am not very familiar with gradle configuration. what is the meaning of  "unsafe" in the config option name?

Reply:
Considered "unsafe" to use. It's a property name defined by Gradle.